### DISCUSSION TITLE
monai.transforms.RandZoom is slow

### DISCUSSION THREAD

**Discussion by NinelieAimer** [Project-MONAI/MONAI#7259] [Q&A]
When augmenting 3D medical images with RandZoom in MONAI, I'm experiencing slow processing speeds. Despite using CacheDataset, the GPU utilization remains low. However, this issue doesn't occur with other augmentation methods.

**Comment by KumoLiu** [comment]
Hi @NinelieAimer, it's reasonable that `RandZoom` (including interpolate) slower than other augmentation (such as RandFlip). And `CacheDataset` will not cache random transforms.
https://github.com/Project-MONAI/MONAI/blob/8e134b8cb92e3c624b23d4d10c5d4596bb5b9d9b/monai/data/dataset.py#L704

**Reply by NinelieAimer** [reply]
Thank you for your reply. Are there any methods to expedite the training process? The current duration is quite lengthy.

**Reply by KumoLiu** [reply]
Hi @NinelieAimer, you could refer to this [tutorial](https://github.com/Project-MONAI/tutorials/blob/main/acceleration/fast_training_tutorial.ipynb) such as using multi-threads `ThreadDataLoader`.
